src/isPalindrome.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+export default function isPalindrome(str) {
+  let noPunct = str.replace(/[.,\/#!$%\^&\*;:{}=\-_`~()]/g,"");
```

Comment:
Use `const` if you're not planning on changing the value of the variable you're initializing.